src/Handler/ExceptionHandler.php

```diff
@@ -104,6 +104,8 @@ public function __invoke(
             $handler = $this->handler($type);
             $this->whoops->pushHandler($handler);
 
+            $this->whoops->writeToOutput(false);
+            $this->whoops->allowQuit(false);
```

Comment:
This is already defined in [the configuration](https://github.com/equip/framework/blob/93aa4fa2e6a08cfddd2607caab11f2931291969d/src/Configuration/WhoopsConfiguration.php#L31-L32), why does it need to be applied again?